docs/api/index.html

```diff
@@ -2602,7 +2592,8 @@ <h3 id="planetscale.com/v2.VitessBackupScheduleStatus">VitessBackupScheduleStatu
 </td>
 <td>
 <em>(Optional)</em>
-<p>A list of pointers to currently running jobs.</p>
+<p>A list of pointers to currently running jobs.
+This field is deprecated and no longer used in &gt;= v2.15. It will be removed in a future release.</p>
```

Comment:
Is this file generated? I'm not repeating my comments from api.md here.

Reply:
`docs/api.md` and `docs/api/index.html` are both generated using the same source.